modules/oe_whitelabel_helper/src/Plugin/Block/ContentBannerBlock.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_whitelabel_helper\Plugin\Block;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Session\AccountInterface;
+
+/**
+ * Provides a view mode block.
+ *
+ * @Block(
+ *   id = "oe_w_content_banner",
+ *   admin_label = @Translation("Content banner"),
+ *   context_definitions = {
+ *     "node" = @ContextDefinition("entity:node", label = @Translation("Node"))
+ *   },
+ *   provider = "node",
+ * )
+ */
+class ContentBannerBlock extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function blockAccess(AccountInterface $account): AccessResult {
+    /** @var \Drupal\node\NodeInterface $node */
+    $node = $this->getContextValue('node');
+
+    /** @var \Drupal\Core\Entity\EntityViewModeInterface|null $view_mode_object */
+    $view_mode_object = \Drupal::entityTypeManager()
+      ->getStorage('entity_view_display')
+      ->load('node.' . $node->bundle() . '.oe_w_content_banner');
+
+    return AccessResult::allowedIf($view_mode_object !== NULL);
```

Comment:
I added the access check.

I was wondering if we need to add the node as cacheable dependency. But it seems we don't, because the node is coming from a context, and therefore the caching is already sensitive to that.

I also wondered if we should add a test for the cache tags context. But as there are no cache tags, it would be quite pointless. Also, I did not find any such tests in oe_theme.